ivy/functional/frontends/paddle/nn/functional/activation.py

```diff
@@ -98,3 +98,26 @@ def softsign(
     name=None,
 ):
     return ivy.divide(x, ivy.add(1, ivy.abs(x)))
+
+
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def celu(
+    x,
+    /,
+    *,
+    alpha=1.0,
+    name=None,
+):
+    prod = ivy.multiply(
+        alpha,
+        ivy.subtract(
+            ivy.exp(ivy.divide(x, alpha)),
+            1,
+        ),
+    )
+    ret = ivy.add(
+        ivy.maximum(0, x),
+        ivy.minimum(0, prod),
```

Comment:
Same here, a simple `+` would be more readable

Reply:
Correct 💯